### PR TITLE
Dev modify cnn training param

### PR DESCRIPTION
**Pull request by Flowingsun007**

### CODE REVIEW

File: Classification/cnns/of_cnn_train_val.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def TrainNet():
     else:
         loss = flow.nn.sparse_softmax_cross_entropy_with_logits(labels, logits, name="softmax_loss")
 
-    flow.losses.add_loss(loss)
+    flow.losses.add_loss(flow.math.reduce_mean(loss))
```
Comment (ShawnXuan) [inline Classification/cnns/of_cnn_train_val.py:83]: 分开两行写
loss = flow.math.reduce_mean(loss)
flow.losses.add_loss(loss)

以为fp16的时候可能最好不用reduce mean。

**Review by ShawnXuan — CHANGES_REQUESTED** [review]
其实可以分成两个pr，这次我不纠结这一点了。

**Comment by Flowingsun007** [comment]
> 其实可以分为两个pr，这次我不纠结这一点了。
好的，下次尽量分开pr

**Review by ShawnXuan — APPROVED** [review]
